### PR TITLE
feat(plugin-registry): Update the Openshift...

### PR DESCRIPTION
**Pull request by nickboldt**
Signed-off-by: Igor Vinokur <ivinokur@redhat.com>

### What does this PR do?
Update the Openshift connector plugin to latest version

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-1733

### CODE REVIEW

**Comment by nickboldt** [comment]
replacement for https://github.com/redhat-developer/codeready-workspaces/pull/530 so that the build test will pass (missing secrets when run from developer fork's topic branch)

**Review by svor — APPROVED** [review]

**Review by ericwill — CHANGES_REQUESTED** [review]
Needs to have dependencies updated first, and we should wait until the vscode-commons dependency has been removed

File: dependencies/che-plugin-registry/che-theia-plugins.yaml [hunk 1]
```diff
@@ -259,14 +259,14 @@ plugins:
...
-      revision: v0.1.5
+      revision: v0.2.7
     sidecar:
       image: "registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.10"
       name: vscode-openshift-connector
       memoryLimit: 1500Mi
       cpuLimit: 500m
       cpuRequest: 30m
-    extension: https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-openshift-tools/vscode-openshift-connector-0.1.5.vsix
+    extension: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v31f30bd/vscode-openshift-connector-v0.2.7.vsix
```
Comment (vinokurig) [inline dependencies/che-plugin-registry/che-theia-plugins.yaml:269]:
```suggestion
    extension: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v1cb50f5/vscode-openshift-connector-v0.2.9.vsix
```

**Comment by mohitsuman** [comment]
There will be a new release of the OpenShift Connector `0.2.10` tomorrow. We need to update the PR accordingly. The fix around latest oc will also be included.

**Comment by mohitsuman** [comment]
New release of OpenShift Connector 0.2.10 is now released. Here are the artifacts: https://github.com/redhat-developer/vscode-openshift-tools/releases/tag/v0.2.10

File: dependencies/che-plugin-registry/che-theia-plugins.yaml [hunk 2]
```diff
@@ -259,14 +259,14 @@ plugins:
...
-      revision: v0.1.5
+      revision: v0.2.9
     sidecar:
       image: "registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8:2.10"
       name: vscode-openshift-connector
       memoryLimit: 1500Mi
       cpuLimit: 500m
       cpuRequest: 30m
-    extension: https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-openshift-tools/vscode-openshift-connector-0.1.5.vsix
+    extension: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/v1cb50f5/vscode-openshift-connector-v0.2.9.vsix
```
Comment (svor) [inline dependencies/che-plugin-registry/che-theia-plugins.yaml:269]:
```suggestion
    extension: https://github.com/redhat-developer/codeready-workspaces-vscode-extensions/releases/download/va21efef/vscode-openshift-connector-v0.2.11.vsix
```

File: dependencies/che-plugin-registry/che-theia-plugins.yaml [hunk 3]
```diff
@@ -259,14 +259,14 @@ plugins:
     extension: https://github.com/che-incubator/che-theia-openshift-auth/releases/download/0.0.1/che-openshift-authentication-plugin-0.0.1.vsix
   - repository:
       url: 'https://github.com/redhat-developer/vscode-openshift-tools'
-      revision: v0.1.5
+      revision: v0.2.9
```
Comment (svor) [inline dependencies/che-plugin-registry/che-theia-plugins.yaml:262]:
```suggestion
      revision: v0.2.11
```

**Comment by nickboldt** [comment]
backported to 2.13 as 57c110e7

**Pull request merged**